src/github/middleware.ts

```diff
@@ -82,16 +82,12 @@ export default (
 		// State change actions are allowed because they're one-time actions, therefore they won’t cause a loop.
 		if (
 			context.payload.sender.type === "Bot" &&
-			!isStateChangeAction(context.payload.action)
+			!isStateChangeOrDeploymentAction(context.payload.action)
 		) {
-			context.log(
-				{
-					noop: "bot",
-					botId: context.payload.sender.id
-				},
-				"Halting further execution since the sender is a bot and action is not a state change"
-			);
-			return;
+			if (!isStateChangeOrDeploymentAction(context.name)) {
```

Comment:
shouldn't these 2 conditions be merged into a single if statement? or is there something missing after the second if?